charts/datahub/subcharts/acryl-datahub-actions/values.yaml

```diff
@@ -38,6 +38,8 @@ service:
 extraLabels: {}
   # owner: myteam
   # environment: test
+annotations: {}
```

Comment:
Just a minor request, since we have `extraLabels` above, it might make more sense to call this `extraAnnotations`. I know that we don't have any existing ones today, but perhaps in the future.